tests/Aspire.Azure.AI.OpenAI.Tests/Aspire.Azure.AI.OpenAI.Tests.csproj

```diff
@@ -5,6 +5,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <None Include="$(RepoRoot)src/Components/Aspire.Azure.AI.OpenAI/ConfigurationSchema.json" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
(nit) can we use consistent slashes in the .csproj? Everywhere else uses `\`, see the 2 lines below.